schemas/v1/list-role-ids-response.yml

```diff
@@ -0,0 +1,21 @@
+$schema:  http://json-schema.org/draft-06/schema#
+title:                      "Get Role Ids Response"
+description: |
+  Get a list of role IDs
```

Comment:
Let's add some text here:

> If no limit is given, the roleIds of all roles are returned.  Since this list may become long, callers can use the `limit` and `continuationToken` query arguments to page through the responses.